bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/device/MessageHandler.java

```diff
@@ -971,16 +971,20 @@ public int transform(int raw) {
             return (raw);
         }
 
-        private int extractValue(Msg msg) throws FieldException {
+        private int extractValue(Msg msg, int group) throws FieldException {
             String lowByte = getStringParameter("low_byte", "");
-            if (lowByte == "") {
+            if (!lowByte.equals("")) {
```

Comment:
I wish I could answer that for you! I've put using Eclipse on hold temporarily. If you could make the change and get it into a nightly build I'd be more than happy to let you know.